src/main/common/draft/draftMiddleware.ts

```diff
@@ -1,40 +1,77 @@
 import * as express from 'express'
 import DraftStoreClient from 'common/draft/draftStoreClient'
+import { Draft, DraftDocument } from 'models/draft'
+import { DraftStoreClientFactory } from 'common/draft/draftStoreClientFactory'
 
-export class DraftMiddleware<T> {
-  client: DraftStoreClient<T>
+/**
+ * Extracts external ID from paths in the '/case/<external-id>.*' format
+ */
+function extractExternalId (path: string): string {
+  const pattern = /\/case\/([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})\/.*/
```

Comment:
This is already done in the claimMiddleware, this should be extracted somewhere common and then reused in both places

Reply:
Missed that one, good idea to reuse. Created `UUIDUtils.extractFrom` for that purpose :)